functions/src/client/ping.ts

```diff
@@ -0,0 +1,18 @@
+import { get } from 'request-promise-native';
+
+export interface PingResponse {
+  statusCode: number;
+  body: string;
+}
+
+export async function Ping<T>(uri: string, token: string): Promise<T> {
+  return <T> await get({
```

Comment:
In the monitor create form it is possible to add a `POST` as this is not supported for now, we should remove `POST` from the html dropdown and only have `GET` as an option

Reply:
I didn't understand "In the monitor create form it is possible to add a POST as this is not supported for now"

Reply:
The ping here is using a `get`, we would need to add functionality for `post` in the **ping client** to support this. As the **ping client** here only supports `get`, I think we remove the `post` option from the form for now and in a future ticket support `post`

Reply:
I have removed the POST from dropdown though